### ISSUE TITLE
Player standing on bearings server crashes

### ISSUE THREAD

**Issue by DSword91** [Fabricators-of-Create/Create#1271]
### Describe the Bug

The player is standing in the create: station_ On the description, a server crash occurred immediately, with a stationary driven by a water wheel_ Containment

### Reproduction Steps

1.Make a power bearing and let it run properly
2.Stand up
3.Server crash

### Expected Result

The server should not crash

### Screenshots and Videos

![image](https://github.com/Fabricators-of-Create/Create/assets/58614972/880b6a0e-1b51-4340-b342-d17a3afd4d5d)

### Crash Report or Log

https://pastebin.com/t5zVKzgM

### Operating System

server:ubuntu 20.04.4 , Client:windows 10

### Mod Version

0.5.1d

### Minecraft Version

1.20.1

### Other Mods

There are many, and they are listed in the crash log. Thank you

**Comment by drwoops** [comment]
what i see is not what you describe: i see the known issue with a deployer killing a mob and then causing a crash.

are you using a fork of create? currently create 0.5.1d sodium 0.4.10 fabric api 0.88.1 iris 1.6.4 and indium 1.0.21 are the precise versions you want. hephaestus is currently incompatible with create (it uses a modified version of porting lib). also check that your version of steam n rails is correct according to this table: https://github.com/Layers-of-Railways/Railway/wiki

**Comment by DSword91** [comment]
Yes, I used a branch: https://github.com/DM-Earth/create-fabric-dme-edition. To address compatibility issues with Hephaestus

And my Steam n Rails version is correct

I once encountered a similar problem on 1.18.2, which was a hydraulic powered logging machine. When I stood on it, the server would crash (the error report had already been deleted)
And it will also crash when starting this device (only once)

**Comment by drwoops** [comment]
if you are using a branch you should report the issue to them.  please close this issue as it is not related to the official create release.

**Comment by DSword91** [comment]
But I had similar issues using official mechanical power before, but my error report was deleted long ago.
This is my second time encountering this problem, that's why I came here to ask the question.
So should I reproduce the official mechanical power error report and upload it again？

**Comment by drwoops** [comment]
yes, please. close this issue and open a new one using the official release.

**Comment by TropheusJ** [comment]
duplicate of #1096
